### DISCUSSION TITLE
Multiple buzzers

### DISCUSSION THREAD

**Discussion by Ant10117** [arendst/Tasmota#16777] [Q&A] [answered]
Is it possible to have multiple buzzers?

I tried setting GPIO14 & 15 to buzzer and using buzzer1 & buzzer2 but both operate the first instance. I like using buzzer for a simple LED indicator as you can set the pulse/flash etc.

**Comment by barbudor** [comment, marked as answer]
No. Only one buzzer.
You can set as a Relay and see the `Blink` command

**Reply by Ant10117** [reply]
Thanks. Using blink is more complex as I can only send a single MQTT message (unless I use rules) so buzzer works because I can specify the parameters. I am surprised it doesn't work the same way that relays and switches function.

**Reply by sfromis** [reply]
Strictly speaking, there is no `Blink` command, but you have `Power blink`, `BlinkTime` and `BlinkCount`. You do not need more than one command via MQTT to initiate blinking.

**Reply by Ant10117** [reply]
That's true but using "blink" you can't change the parameters without sending multiple messages, which you can using buzzer. I just found it strange that configuring multiple buzzers didn't work the same as other device types so wanted to confirm it wasn't a me problem :).

**Reply by sfromis** [reply]
You **can** do it all in one message, otherwise I'd not have made that statement.

Something like this command in a single message can work:
`Backlog0 BlinkCount 10; BlinkTime 10; Power Blink`

More generally, you can only have more than one usage for a component type dropdown selection if you get the opportunity to select a channel number to the right. Without this, only a single pin is supported.

**Reply by Ant10117** [reply]
I guess there's a limitation on my MQTT client as I tried almost that exact string and it failed. It may be the semi-colon or maybe the string length, now I know it's possible I will do some more testing. Thank you!

**Reply by sfromis** [reply]
What should work is a topic like `cmnd/yourdevice/Backlog0` and the payload `BlinkCount 10; BlinkTime 10; Power Blink`

Note that many command line environments will have the shell using characters like `;` for other purposes, in such situations it generally helps to wrap the string in quotes.
